projects/jbig2dec/build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+find /src | grep -v '^/src/honggfuzz/' | grep -v '^/src/afl/' | grep -v '^/src/libfuzzer/'
```

Comment:
What's going on here?

Reply:
That is debugging information that I meant to remove before submitting. :) This will make the log files slightly longer, and if you want me to I can submit another change to remove this line, else I'll remove it next time I update the jbig2dec project files.

Reply:
I'd prefer it were removed but if you don't remove it no big deal.

Reply:
Alright, I created pull request 2831 to resolve this.